src/main/java/it/ipzs/fedauthority/oidclib/OidcConfig.java

```diff
@@ -0,0 +1,448 @@
+package it.ipzs.fedauthority.oidclib;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import it.ipzs.fedauthority.oidclib.schemas.OIDCProfile;
+
+@Configuration
+@ConfigurationProperties(prefix = "oidc")
```

Comment:
openid federation is not oidc anymore, please align this

Reply:
Modified, but these properties were configured from a previous version of a library shared in the last projects (i.e. eudi-wallet-it-openid-credential-issuer), there will be a refactor about it in a next version.

Reply:
Updated